### ISSUE TITLE
how to deal with the missing data?

### ISSUE THREAD

**Issue by vision-zhao** [liyaguang/DCRNN#18]
there are so many missing data and nil data in the METR.h5, so I'm wondering how you deal with it when you are producing your result?

I will be deeply grateful to you!

**Comment by liyaguang** [comment]
Currently, we just ignore the missing value when computing the training loss. 
You may also refer to [1][2] for dealing with missing data in time series data.
[1] Zhengping Che, Sanjay Purushotham, Kyunghyun Cho, David Sontag, and Yan Liu, Recurrent Neural Networks for Multivariate Time Series with Missing Values,  Scientific Reports (SREP), 8(1):6085, 2018.
[2] Dingxiong Deng, Cyrus Shahabi, Ugur Demiryurek, Linhong Zhu, Rose Yu and Yan Liu, Latent Space Model for Road Networks to Predict Time-Varying Traffic, International Conference on Knowledge Discovery and Data Mining (KDD), 2016.

**Comment by RobinLu1209** [comment]
Hi, I still have some problems about missing data. It's a good choice to ignore the missing value when computing the training loss, but what's your action to train? There will be data mutation and abnormality in training.
